src/domain/event/similarEvents/__tests__/SimilarEvents.test.tsx

```diff
@@ -60,17 +68,24 @@ afterAll(() => {
   clear();
 });
 
-test('should render similar event cards', async () => {
+const waitForComponentToBeLoaded = async (render: () => CustomRenderResult) => {
```

Comment:
can't quite see the benefit passing the render as a callback here

Reply:
```
render(<SimilarEvents event={event} />, { mocks })
await waitForComponentToBeLoaded();
```

that seems easier to me

Reply:
At some point it was benefical but now unnedessary complex as nothing is done with rendering result inside the method. I can fix it.